### DISCUSSION TITLE
Martin is going down

### DISCUSSION THREAD

**Discussion by HiralBabariya** [maplibre/martin#1403] [Q&A]
I am using martin v0.5.0 with postgresql. I have large data table for create tiles. Now, When i started requesting martin for tiles it is giving 2 or3 request fast than after it's loading very much.

Please help me out. My product is on production.

I found main error on log is : 

Unable to get tile 11/1012/673 from land_registry_aug_2023: connection closed
Error in response: "Unable to get tile 11/1012/673 from land_registry_aug_2023: connection closed"

**Comment by 0x-a6** [comment]
Your SQL server is overloaded, reduce your connection count in your config
